src/main/resources/welshJurisdictionLookup.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Civil": "Llys Sifil",
```

Comment:
I'm wondering if this should be part of listLookup rather than a seperate file.

It does mean a bit of duplication, bit feels a bit cleaner as they're so closely related

Reply:
Removed welshJurisdictionLookup and added it to listLookup